.goreleaser.yml

```diff
@@ -64,9 +95,9 @@ nfpms:
       rpm:
         replacements:
           amd64: x86_64
-        file_name_template: "influxdb2-{{ .Version }}.{{ .Arch }}{{if .Arm}}{{ if eq .Arm \"5\" }}el{{end}}{{ end }}{{if .Arm}}{{ if eq .Arm \"6\" }}hf{{end}}{{ end }}"
+        file_name_template: "influxdb2-{{ .Version }}.{{ .Arch }}{{ if .Arm }}v{{ .Arm }}hf{{ end }}"
```

Comment:
TODO before merge: Make sure these template changes don't break anything.

* The case for 5 was removed because we're not building for `GOARM=5`.
* `GOARM=6` should have the same output.
* `GOARM=7` will now output with an `hf` suffix

Reply:
Decision: keep v6 as `armhf` suffix, use `armv7hf` for v7